deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -889,6 +889,9 @@ end}.
 
 {mapping, "channel_max", "rabbit.channel_max", [{datatype, integer}]}.
 
+{mapping, "channel_max_per_node", "rabbit.channel_max_per_node",
+          [{datatype, integer}, {validators, ["non_negative_integer"]}]}.
```

Comment:
This should be a positive integer, setting this value to 0 does not make much sense (unless you want to effectively block AMQP 0-9-1 clients specifically, which I don't think is what we're after here).

Reply:
Good point. Actually, 0 here means 'infinite', so I might just change that to be the default for readability.

Reply:
Ah, right. Then this validator makes sense. As for the default, we use `cuttlefish:unset/0` as a return value when we want to default in the code to kick in.

Reply:
Done, changed 0 -> infinity

Reply:
OK, I think it makes more sense than `0`, and for new settings we can adopt this new value. Most users who want no limits probably won't set it in the first place.